Components/HomeSidebar.tsx

```diff
@@ -20,16 +19,20 @@ const HomeSidebar = ({ pageNumber = 1 }: any) => {
   };
 
   return (
-    <div className="fixed top-0 flex  flex-col  p-4 w-fit h-screen justify-between ">
-      {/* <LogoWithName width={40} height={40} /> */}
-      <div className="flex flex-col space-y-4">
+    <div className="absolute top-0 flex  flex-col  p-4 justify-between w-[304px] h-screen left-[1px] bg-gradient-to-b from-[#e2e9f1] to-white">
+      {/* Top Part  */}
+      <LogoWithName width={40} height={40} />
+
+      {/* Center Part  */}
+      <div className="flex flex-col items-start gap-[24px] p-0 w-[304px] h-[360px] top-[207px] left-[calc(50%-152px)] absolute">
```

Comment:
What are these changes? There is nothing about these in the PR description. Please, always try to explain a little bit about your changes in the PR description.